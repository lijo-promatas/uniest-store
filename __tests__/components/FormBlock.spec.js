import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import FormBlock from '../../src/components/FormBlock';

it('renders <FormBlock /> component correctly', () => {
  const tree = renderer.create(<FormBlock title="test" noContainerStyle>Test</FormBlock>).toJSON();
  expect(tree).toMatchSnapshot();
});
