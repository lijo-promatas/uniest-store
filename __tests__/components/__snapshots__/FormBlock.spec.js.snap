// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <FormBlock /> component correctly 1`] = `
<View
  style={
    Array [
      undefined,
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    TEST
  </Text>
  <View>
    Test
  </View>
</View>
`;
